app/controlplane/plugins/core/slack-webhook/v1/slack_webhook.go

```diff
@@ -0,0 +1,208 @@
+//
+// Copyright 2023 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package slack_webhook
```

Comment:
I'd just call it slack. That would remove the lint warning and allow you to import it properly in `plugins` without the alias

```
	slack "github.com/chainloop-dev/chainloop/app/controlplane/plugins/core/slack-webhook/v1"
```